src/index.ts

```diff
@@ -148,6 +148,19 @@ export class Collection<K, V> extends Map<K, V> {
 		return arr.slice(-amount);
 	}
 
+	/**
+	 * Identical to [Array.at()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at).
+	 * Returns the item at a given index, allowing for positive and negative integers.
+	 * Negative integers count back from the last item in the collection.
+	 * @param {number} index The index of the element to obtain
+	 * @returns {*}
+	 */
+	public at(index: number): K | undefined {
+		const arr = [...this.keys()];
```

Comment:
```suggestion
		index = Math.floor(index);
		const arr = [...this.keys()];
```
If you want to be [spec-compliant](https://tc39.es/ecma262/multipage/indexed-collections.html#sec-array.prototype.at), then the index should be floored. Otherwise, you can ignore this.

Reply:
Would like to hear from @iCrawl on this one